riski-extractor/Dockerfile

```
@@ -1,5 +1,17 @@
-# Smallest ubi10 image
-FROM registry.access.redhat.com/ubi10/ubi-micro:latest
+
+FROM  registry.access.redhat.com/ubi10/ubi-minimal:latest
+
+USER root
+
+RUN microdnf install glibc-all-langpacks -y && \
+    microdnf clean all
+
```

```suggestion
USER 1000
```

switch back to non root user